Gemfile.lock

```diff
@@ -150,7 +150,7 @@ GEM
     e2mmap (0.1.0)
     erubi (1.9.0)
     erubis (2.7.0)
-    exercism-config (0.47.0)
+    exercism-config (0.48.0)
```

Comment:
I needed to update this in order to make the new `git-server` work.

Reply:
I have this in my reflog! But for some reason it never made it upstream. Sorry dude.

<img width="452" alt="Screenshot 2020-10-28 at 13 36 12" src="https://user-images.githubusercontent.com/286476/97443244-bb7c4c80-1922-11eb-9da9-c634497ef381.png">